### ISSUE TITLE
Granada support (All existing features, new protocol features not tracked in this issue)

### ISSUE THREAD

**Issue by Innkst** [ecadlabs/taquito#900]
<details>
	<summary>Integration with the Granada protocol</summary>

## Tasks to make sure existing projects using v9 continue to work with v10/grandanet

- [ ] Setup `grandanet` public nodes (https://api.tez.ie/rpc/florencenet)
- [ ] Add `grandanet` to the keygen API
- [ ] Add `grandanet` to integration tests
- [ ] Add `grandanet` constants ` packages/taquito/src/constants.ts`
- [ ] Deploy lambda-view contract
- [ ] Add new protocol to the `ContractView` class for [lambdaView](https://github.com/ecadlabs/taquito/blob/master/packages/taquito/src/contract/contract.ts#L134)
- [ ] Deploy `knownContract` for tests
- [ ] Delploy `knownBigMapContract`
- [ ] Update known baker address `tz1cjyja1TU6fiyiFav3mFAdnDsCReJ12hPD`
- [ ] Update estimate values form `grandanet` 
- [ ] Add the protocol to the michel-codec package
- [ ] Add the protocol to the tzip16 package, in the michelson-storage-view.ts
- [ ] Add GitHub Actions to run grandanet tests from CI
</details>
<details>
	<summary>Integration with the galphanet protocol (refer to #817) - anything needs to get merged??</summary>

## Tasks to make sure existing projects using v9 continue to work with v10/galphanet

- [ ] Setup `galphanet` public nodes (https://api.tez.ie/rpc/falphanet)
- [ ] Add `galphanet` to the keygen API
- [ ] Add `galphanet` to integration tests
- [ ] Add `galphanet` constants ` packages/taquito/src/constants.ts`
- [ ] Deploy lambda-view contract
- [ ] Add new protocol to the `ContractView` class for [lambdaView](https://github.com/ecadlabs/taquito/blob/master/packages/taquito/src/contract/contract.ts#L134)
- [ ] Deploy `knownContract` for tests
- [ ] Delploy `knownBigMapContract`
- [ ] Update known baker address `tz1cjyja1TU6fiyiFav3mFAdnDsCReJ12hPD`
- [ ] Update estimate values form `galphanet` 
- [ ] Add the protocol to the michel-codec package
- [ ] Add the protocol to the tzip16 package, in the michelson-storage-view.ts
- [ ] Add GitHub Actions to run galphanet tests from CI
</details>

## Assess the impact of New Features/Functionality in Granada and create follow-up issues to add support for them:
- Emmy*: most likely has impact on the estimations
- Liquidity Baking
   - [ ] Subsidy has to be added to all operations. Estimations impact.
- Gas improvements
- RPC changes
   - [ ] Create a follow-up issue to download bigmaps in parallel
   - [ ] Create a follow-up issue to add/remove any new RPCs and updates to the existing ones.
- Metadata changes
- Minor changes

## Resources
- 010 Granada Protocol Change Log https://tezos.gitlab.io/protocols/010_granada.html

## Misc notes

To get the list of protocol hash's supported `tezos-admin-client list protocols`
`PsrsRVg1Gycjn5LvMtoYSQah1znvYmGp8bHLxwYLBZaYFf2CEkV`    proto hash for `falphanet`, short version `PsrsRVg1`

Code diff for similar work [here](https://github.com/ecadlabs/taquito/commit/14dad3b55955e7a3b1cfee5a89c425097315386e).

take this opportunity to add more steps and refine our [wiki](https://www.notion.so/ecad/Adding-a-new-protocol-07496f004819476599a08b76e8464c14).
